css-modules/authoring.less

```diff
@@ -0,0 +1,88 @@
+.authoring {
+  font-size: 16px;
+  padding: 20px;
+
+  hr {
+    width: 300px;
+    margin: 10px 0;
+    border: 1px solid #eee;
+  }
+
+  h2 {
+    margin-bottom: 10px;
+  }
+
+  h3 {
+    margin-top: 20px;
+  }
+
+  iframe {
+    border: 1px solid #aaa;
+  }
+
+  .inlineInput {
+    display: inline-block;
+    margin-left: 10px;
+    margin-right: 10px;
+  }
+
+  .section {
+    margin-left: 10px;
+  }
+
+  .finalUrl {
+    border: 1px solid #999;
+    padding: 10px;
+    border-radius: 3px;
+  }
+
+  .autocompleteLabel {
+    font-size: 14px;
+    color: #444;
+  }
+}
+
+// text input theme
+.input {
+  width: 250px;
+  margin: -5px 0 -17px 0;
+
+  .inputElement {
+    padding: 3px 0;
+  }
+
+  .label {
+    color: #666;
+    top: 24px;
+  }
+
+  .filled ~ .label {
+    color: #555;
+  }
+}
+
+// checkbox theme
```

Comment:
I read up on CSS modules - having scoped CSS does seem really nice, I'll try it in my next commit. I guess if the toolbox components are written with css modules, the class names all end up gibberish, so you need to use css modules to style them?

Reply:
Yes, exactly. Sometimes it's enough to assign className to the top component and then try to use element matches, but quite often it's annoying and I reached the point that I couldn't do that. With CSS modules, you just specify class names that are described in react-toolbox docs and you can theme all the elements.

It makes sense and it won't hurt our code either. No need to care about unique class names.